numpy/core/shape_base.py

```diff
@@ -179,6 +171,99 @@ def atleast_3d(*arys):
         return res
 
 
+def atleast_nd(ary, n, pos=0, inplace=False):
+    """
+    View input as array with at least `n` dimensions.
+
+    Parameters
+    ----------
+    ary : array_like
+        The input array. Non-array inputs are converted to arrays.
+        Arrays that already have `n` or more dimensions are preserved.
+    n : scalar
+        The minimum number of dimensions required.
+    pos : int, optional
+        The index of the new dimensions. All new dimensions are inserted
+        into the specified position. 0 and -1 always guaranteed to work.
```

Comment:
Can you clarify what valid positions are? Something like "specified position of axes on the input array"?